client/src/Components/Alerts/AlertList.js

```diff
@@ -1,17 +1,31 @@
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
-import webHost from '../../config/web';
+import { useHistory} from 'react-router-dom';
 
-import Card from '../../UIElements/Card'
+import webHost from '../../config/web';
+import api from '../../config/web';
+import Card from '../../UIElements/Card';
 
 const AlertList = () => {
   const [list, setList] = useState([]);
   const [loading, setLoading] = useState(true);
+  const history = useHistory();
 
+  const deleteHandler = (id) => {
+    //delete alert using the id
+    axios.delete(`${api.URI}/${id}`)
+      .then((response) => {
+        console.log(response.data);
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+    //reload page after deleting alert
+    window.location.reload();
```

Comment:
if this reload needs to happen on delete completion it should happen within the `then` function, e.g.:

```
axios.delete({URL})
.then((response) => {
// do reload now
})
.catch((error) => {
// handle error here (pop up message saying something like "Unable to delete alert. Reason ${error.message}")
})

Reply:
Thank you for the advice!